frontends/web/src/api/exchanges.ts

```diff
@@ -33,6 +33,12 @@ export const getExchangesByRegion = (code: string) => {
   };
 };
 
+
+export const getSellExchangesByRegion = (): Promise<ExchangeRegionList> => {
```

Comment:
I think that we can avoid this endpoint, at least for now. Pocket sell feature will be available in every region where buy is available.

Reply:
Ah, good to know. Okay!